assets/cnv/11_cnv_sample.sh

```diff
@@ -12,7 +12,7 @@ ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@lb-0.${TF_V
 
 ## Stage a Windows 2019 image from Vagrant on you bastion/LB
 
-version=`curl -L -H "Accept: application/json" https://app.vagrantup.com/peru/boxes/windows-server-2019-standard-x64-eval | jq -r '.versions[0].version'`
+version=`curl -fsSL -L -H "Accept: application/json" https://app.vagrantup.com/peru/boxes/windows-server-2019-standard-x64-eval | jq -r '.versions[0].version'`
```

Comment:
i think you can remove the `-L` since it's in the `-fsSL` now. not a big deal, though.